Fix some typos in docs, user-facing content but also comments (#2376)
canonical/subiquity: doc/howto/report-bugs.rst

    apport-cli /path/to/report.crash

> To run ``apport-cli`` in the installer environment, switch to a shell. This way, ``apport`` can not open a browser for you to complete the report. Instead, it provides a URL for completing the report, which you can do on another computer.

Reviewer: This is better, but the "can not" bit feels awkward.  What do you think about:
```suggestion
To run ``apport-cli`` in the installer environment, switch to a shell. This inhibits ``apport`` from opening a browser for you to complete the report. Instead, it provides a URL for completing the report, which you can do on another computer.
```

Author: Thanks! Yes, this is better. Applied
